Userland/Libraries/LibAudio/Sample.h

```diff
@@ -173,6 +179,13 @@ struct Sample {
         return *this;
     }
 
+    constexpr Sample& operator-=(Sample const& other)
+    {
+        left -= other.left;
+        right -= other.right;
+        return *this;
+    }
```

Comment:
Why have two commits adding generic operators to `Sample`? I'd combine this with the first commit (the one adding multiply/divide)

Reply:
Right, I'd rather do that than to split the commit up after the fact :^)